message-commands/index.js

```diff
@@ -0,0 +1,32 @@
+const { createGraph } = require("../graphviz-helper")
+
+
+const lookForCommand = (message) => {
+    // 835044781839089664 is the valorant channel
+	// if(message.channelId === '835044781839089664' && /lf\dm/.test(message.content.toLocaleLowerCase())){
+	// 	let rawdata = fs.readFileSync(path.join(__dirname, 'looking-for-queue.json'))
+	// 	let lookingForQueue = await JSON.parse(rawdata);
+	// 	let mentionString = ''
+
+	// 	if(lookingForQueue.length === 0) {
+	// 		message.channel.send('No one is on deck')
+	// 		return
+	// 	}
+	// 	for(index in lookingForQueue){
+	// 		mentionString += ` <@${lookingForQueue[index]}>`
+	// 	}
+	// 	mentionString += ' WAKE UP BITCHES ITS VALOTIME'
+	// 	message.channel.send(mentionString)
+
+	// }
+}
+
+const messageCommandProvider = (message) => {
+    lookForCommand(message)
+    if(message.content=='graph'){
```

Comment:
this is so that when someone types the word graph it will trigger this message handler?

Reply:
yeah but its not permanent
this is just a placeholder